docs/disk_failure.md

```diff
@@ -0,0 +1,48 @@
+# Disk failure
+
+The `diskFailure` field offers a way to apply a disk failure on a specific process and path.
+
+## How it works
+
+The disruption runs an eBPF disk failure program used to intercept system calls and inject errors. 
+It is used to prevent certain processes from accessing certain files. 
+It defines a target process and a filter path, and if the process is trying to 
```

Comment:
Could you explain how matching the filter path works a bit more? Is it a strict prefix match? Does it take wildcards?